.ruby-version

```diff
@@ -0,0 +1 @@
+ruby-2.6.6
```

Comment:
Ruby 2.7 gives warnings about kwargs, but these warnings are no longer valid as Ruby 3.0 changed how they work, so I've just stuck with 2.6.6 for now, in line with all the other Ruby codebases (e.g. Test Runner)